avocado/utils/asset.py

```diff
@@ -684,9 +685,15 @@ def remove_asset_by_path(cls, asset_path):
 
         :param asset_path: full path of the asset file.
         """
-        os.remove(asset_path)
-        filename = f"{asset_path}-CHECKSUM"
-        os.remove(filename)
+        try:
+            file_path = pathlib.Path(asset_path)
+            file_path.unlink()
+            checksum_path = pathlib.Path(f"{asset_path}-CHECKSUM")
+            checksum_path.unlink()
+        except FileNotFoundError:
```

Comment:
The exception handling is a great idea, and it is something which is needed here. Good job.